.github/workflows/linters.yml

```diff
@@ -0,0 +1,48 @@
+name: Linters
+
+on: pull_request
+
+env:
+  FORCE_COLOR: 1
+
+jobs:
+  lighthouse:
+    name: Lighthouse
+    runs-on: ubuntu-18.04
```

Comment:
- [ ] It would be better if you give more explanation of missing things in the design instead of just asking to match the design 
![image](https://user-images.githubusercontent.com/74744787/160492943-37c7f6ab-d50a-47e4-bc90-1a17efa47b0a.png)